### ISSUE TITLE
Problem with documentation - file not found

### ISSUE THREAD

**Issue by MikeFalowski** [sardana-org/sardana#964]
Sardana docs normally opens on `https://sardana.readthedocs.io/en/stable` but when I try to change a branch or select an element in search list it goes to `https://sardana-controls.org/en/stable` and then:
![image](https://user-images.githubusercontent.com/25459174/47002513-7a46e480-d12d-11e8-99b6-593394c9f30f.png)

**Comment by reszelaz** [comment]
Since #830 we don't rely anymore on RTD for building docs, now we build it directly on travis. The working URLs are https://www.sardana-controls.org and https://sardana-controls.org. BTW I think we never announced directly the *readthedocs.io* sardana docs URLs.

**Comment by MikeFalowski** [comment]
Thanks Zbyszek for clarification. I'm closing the issue.

**Comment by reszelaz** [comment]
Actually there is an issue with the documentation. Initially I attributed this problem to the URL including the "readthedocs": https://sardana.readthedocs.io/en/stable which was never announced as sardana docs site. But actually the https://sardana-controls.org/en/stable which used to work does not work anymore.

This issue appeared after migrating the docs from RTD to GH Pages. It looks like there is no easy way to get it solved but we submitted a question to the GH support and will keep you updated.

Meanwhile I updated the [sardana-training materials](https://github.com/sardana-org/sardana-training/commit/27885bc64df9aa57b74306af5c6c03cb3d88b7c6) to use the new URL.
I used the following bash command to do it automatically:
```
zreszela@pc255:~/workspace/sardana-training (master)> find . -type f -name "*.ipynb" | xargs sed -i "s/\/en\/latest//g"
```
